WAKU.md

```diff
@@ -0,0 +1,17 @@
+# Waku spec support
+
+*Last updated February 14, 2020*
+
+status-nim client of Waku is spec compliant with [Waku spec v0.3](https://specs.vac.dev/waku.html) with the exception of:
+- It doesn't support all the MUST packet codes (!)
```

Comment:
```suggestion
- It doesn't support all the MUST packet codes (WIP status-im/status-go#1853)
```

Reply:
https://github.com/status-im/status-go/issues/1853 doesn't address any of the MUST have packets:

```
Name | Int Value
-- | --
Status | 0
Messages | 1
PoW Requirement | 2
Bloom Filter | 3
```

We have to be more precise about these things.

Reply:
I though `Topic interest` is in a group of must packets but apparently it is not. So the current Waku implementation supports all required packets.